scripts/integrate_ephemeris.jl

```diff
@@ -1,35 +1,115 @@
-const bodyind = 1:(11+16) #1:(11+nast) # body indices in output
-
-# integration parameters
-const order = 25
-const abstol = 1.0E-20
-
-#integrator warmup
-PlanetaryEphemeris.propagate(1, jd0, nyears, output=false, dense=dense, dynamics=dynamics, nast=nast, quadmath=quadmath, bodyind=bodyind, order=order, abstol=abstol)
-println("*** Finished warmup")
-
-# perform full integration
-PlanetaryEphemeris.propagate(maxsteps, jd0, nyears, dense=dense, dynamics=dynamics, nast=nast, quadmath=quadmath, bodyind=bodyind, order=order, abstol=abstol)
-println("*** Finished full integration")
+using ArgParse, PlanetaryEphemeris, Dates
+
+function parse_commandline()
+
+    s = ArgParseSettings()
+    
+    # Program name (for usage & help screen)
+    s.prog = "integrate_ephemeris.jl"  
+    # Desciption (for help screen)
+    s.description = "Integrates JPL DE430 Ephemeris" 
+
+    @add_arg_table! s begin
+        "--maxsteps"
+            help = "Maximum number of steps during integration"
+            arg_type = Int
+            default = 1_000_000 
+        "--jd0"
+            help = "Starting time of integration; options are: \"2000-01-01T12:00:00\" or \"1969-06-28T00:00:00\""
+            arg_type = DateTime
+            default = DateTime(2000, 1, 1, 12) # DateTime(1969, 6, 28, 0, 0, 0)
+        "--nyears"
+            help = "Number of years"
+            arg_type = Float64
+            default = 31.0
+        "--dynamics"
+            help = "Dynamical model function"
+            arg_type = Function
+            default = DE430!
+        "--nast"
+            help = "Number of asteroid perturbers"
+            arg_type = Int
+            default = 343 # 16
+        "--order"
+            help = "Order of Taylor polynomials expansions during integration"
+            arg_type = Int
+            default = 25
+        "--abstol"
+            help = "Absolute tolerance"
+            arg_type = Float64
+            default = 1.0E-20
+        "--parse_eqs"
+            help = "Whether to use the taylorized method of jetcoeffs (a-priori faster) or not"
+            arg_type = Bool
+            default = true 
+        "--bodyind"
+            help = "Body indices in output"
+            arg_type = UnitRange{Int}
+            default = 1:(11+16) # 1:(11+nast)
+            # bodyind = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 17, 18, 19, 20, 21, 22, 23, 25, 27, 28, 30, 40, 41, 46, 55, 62, 73, 113, 115, 277, 322] # SS + 25 ast perturbers
+    end
+
+    s.epilog = """
+        examples:\n
+        \n
+        # Multi-threaded\n
+        julia -t 4 --project integrate_ephemeris.jl --maxsteps 100 --jd0 \"2000-01-01T12:00:00\"\n
+        \n
+        # Single-threaded\n
+        julia --project integrate_ephemeris.jl --maxsteps 100 --jd0 "2000-01-01T12:00:00"\n
+        \n
+    """
```

Comment:
Maybe the `\n` newlines and the backslashes before the quotes enclosing the date/time string are not needed when using triple-quotes?
```suggestion
    s.epilog = """
        examples:
        
        # Multi-threaded
        julia -t 4 --project integrate_ephemeris.jl --maxsteps 100 --jd0 "2000-01-01T12:00:00"
        
        # Single-threaded
        julia --project integrate_ephemeris.jl --maxsteps 100 --jd0 "2000-01-01T12:00:00"
        
    """
```

Reply:
Changed.